IssueCreator/IssueCreator/Helpers/ObjectCreator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Octokit;
+
+namespace IssueCreator.Helpers
+{
+    public static class ObjectCreator
+    {
+        public static Issue CreateIssue(int issueNumber, long RepoId)
+        {
+            Issue issue = new Issue();
+
+            // use reflection to set the 2 values
```

Comment:
Uh...  why?  Maybe add a bit more detail in this comment.  😄

Reply:
Didn't see the 20+ ctor for Issue that would solve the same problem.